plugin/dns64/dns64.go

```diff
@@ -69,13 +70,11 @@ func (d *DNS64) Name() string { return "dns64" }
 
 // requestShouldIntercept returns true if the request represents one that is eligible
 // for DNS64 rewriting:
-// 1. The request came in over IPv6 (not in RFC)
+// 1. The request came in over IPv6, unless AllowIPv4 option is not set
```

Comment:
Clearer would be `or the 'allow_ipv4' option is set`. Avoids double negative wording.

Reply:
right 👍 fixed as you suggested